src/namespace-sandbox/namespace-sandbox.c

```diff
@@ -586,7 +586,7 @@ static void SetupDirectories(struct Options *opt, uid_t uid) {
     } while (entry != NULL && entry->pw_uid != uid);
     fclose(passwd);
 
-    if (entry == NULL) {
+    if ((entry == NULL) || strcmp(entry->pw_dir, "/") == 0) {
```

Comment:
Why do we have to fall back to `$HOME` exactly? Obviously, `/` is a valid directory. I think we should rather fix whatever prevents using `/`...

Reply:
Thanks. I fixed the creation of `/`.